docs/dom-testing-library/api-configuration.mdx

```diff
@@ -80,6 +80,11 @@ with a second argument e.g.
 The default value for the `hidden` option used by
 [`getByRole`](queries/byrole.mdx). Defaults to `false`.
 
+### `defaultIgnore`
+
+The default value for the [`ignore` option](queries/bytext.mdx#ignore) used by
+[`getByText`](queries/bytext.mdx). Defaults to `script, style`.
```

Comment:
Now that I think about it: It's not just `*ByText`.This also affects the nodes we print in error messages.

Reply:
@eps1lon do you have a specific wording you'd like me to put here? 👀

Reply:
@eps1lon I gave it a shot. 😅 What do you think?